TravelAgencyService/Models/AgencyModel.cs

```diff
@@ -0,0 +1,50 @@
+using TravelAgencyService.Services;
+
+namespace TravelAgencyService.Models;
+
+public class AgencyModel
+{
+    private readonly List<Destination> _destinations =
+    [
+        new Destination("Spain", 5),
+        new Destination("Ukraine", 3),
+        new Destination("Japan", 4),
+        new Destination("Norway", 2),
+        new Destination("New Zealand", 3)
+    ];
+
+    private ExceptionMgmt _exceptionMgmt;
+
+    public void BookTicket(string destinationName)
+    {
+        var destination = FindDestination(destinationName);
+
+            if (destination == null)
+            {
+                throw new ExceptionMgmt.InvalidDestinationException("Wrong destination. Please, choose the right one.");
```

Comment:
Its not a very convenient way to use Exceptions in C#, just throw with new and thats enough